client/src/components/CatalogMainSection/CatalogMainSection.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Stack, Typography, Container } from '@mui/material';
+import styles from './CatalogMainSection.module.scss';
+
+function CatalogMainSection() {
+  return (
+    <Stack className={styles.mainSection} justifyContent="center">
```

Comment:
you dont need the stack component here as you dont have a stack of elements. you can substitute it with the Box element which by default also translates into a div HTML tag.

Reply:
I used Stack for text positioning convenience. But your comment makes sense. Done